src/main/java/dagger/ComposerSearchComponent.java

```diff
@@ -0,0 +1,10 @@
+package dagger;
+
+import javax.inject.Singleton;
+
+@Singleton
+@Component(modules = { ComposerSearchModule.class })
+public interface ComposerSearchComponent
+{
+    ComposerSearchFrame getComposerSearchFrame();
```

Comment:
`ComposerSearchFrame` is in the default package (it has no package declaration at the top).
You can't import a class from the default package which is why Java doesn't pick it up.
Normally I don't let students create classes in the default package, so this never comes up. 
Move both the Frame and Presenter into it's own package (swing) and it should work now. 

You'll need to make sure that the class name in gradle has the correct package in it or your jar file won't work anymore.